ci/jobs/satellite6-pulp-upgrade.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: satellite6-pulp-upgrade
+    node: f23-vanilla-np
+    wrappers:
+        - ansicolor
+        - ssh-agent-credentials:
+            users:
+                - '044c0620-d67e-4172-9814-dc49e443e7b6'
+    builders:
+        - shell: |
+            set +e
+            echo "Start upgrading Pulp on $INSTANCE_IP..."
+            ssh -o StrictHostKeyChecking=no cloud-user@$INSTANCE_IP exit
+            alias SSH_CLI="ssh -t -t -o StrictHostKeyChecking=no cloud-user@$INSTANCE_IP"
+            SSH_CLI "sudo katello-service stop"
+            SSH_CLI "rpm -qa | grep pulp"
+            SSH_CLI "sudo yum-config-manager --add-repo https://repos.fedorapeople.org/pulp/pulp/beta/2.8/7Server/x86_64/"
+            SSH_CLI "sudo yum repolist enabled | grep pulp"
+            SSH_CLI "sudo yum update -y --nogpgcheck pulp-*"
+            SSH_CLI "rpm -qa | grep pulp"
+            SSH_CLI "sudo katello-service restart"
+            SSH_CLI "sudo -u apache pulp-manage-db"
+            curl https://bootstrap.pypa.io/ez_setup.py -o - | sudo python
+            SSH_CLI "sudo easy_install pip"
+            SSH_CLI "sudo pip install httpie"
+            SSH_CLI "http https://pulp-jenkins-sat6-tester.os1.phx2.redhat.com/pulp/api/v2/status/"
+            UPDATE_PULP_VERSION=`SSH_CLI "rpm -qa | grep pulp-server"`
```

Comment:
Use `$()` instead of backticks for subshells. Like UPDATE_PULP_VERSION="$(SSH_CLI rpm -qa | grep pulp-server)"